### DISCUSSION TITLE
Enable Save button only when there are any changes made

### DISCUSSION THREAD

**Discussion by jacargentina** [jaredpalmer/formik#3291] [Q&A]
I need to avoid sending a POST request when the user has not made any changes to an original documento with lots of fields; only enable that button when any field has changed. Which way would be THE way ?

**Comment by tsekityam** [comment]
Formik has a [dirty](https://formik.org/docs/api/formik#dirty-boolean) flag, you can enable the save button based on that value.

**Reply by jacargentina** [reply]
My form is a document editor, from time to time the user will save, and continue editing. So after document loading dirty is false: great! After editing something, becomes true: great! Button enabled, users saves changes. Then I need it to be false again!!! Here it is not working anymore as I need.
